platooning_strategic/src/platoon_strategic.cpp

```diff
@@ -134,6 +134,10 @@ namespace platoon_strategic
     void PlatoonStrategicPlugin::checkForRightMostLane(const lanelet::BasicPoint2d& current_location)
     {
         auto current_lanelet = wm_->getLaneletsFromPoint(current_location, 1);
+        if (current_lanelet.size()<1)
+        {
+            std::invalid_argument("There are no lanelets in the current location.");
```

Comment:
I think you're missing the `throw` here, but otherwise this looks good to me. Quick tweak and then we should be good to merge, though Sonar Scanner appears to be broken somehow.